doc/source/reference/arrays.dtypes.rst

```diff
@@ -298,7 +307,6 @@ Type strings
     .. admonition:: Example
 
        >>> dt = np.dtype((void, 10))  # 10-byte wide data block
-       >>> dt = np.dtype((str, 35))   # 35-character string
```

Comment:
I'd be tempted to keep this, but using `bytes` instead. Also, `void` is a NameError on the line above?